packages/teams-js/test/internal/validOrigins.spec.ts

```diff
@@ -409,102 +412,185 @@ describe('validOrigins', () => {
       /* eslint-disable-next-line @microsoft/sdl/no-insecure-url */ /* Intentionally using http here because of what it is testing */
       const messageOrigin = new URL('http://teams.microsoft.com');
-      const result = await validateOrigin(messageOrigin);
+      const result = await validateOrigin(messageOrigin, disableCache);
       expect(result).toBe(false);
     });
     it('validateOrigin returns false if first end of origin is not matched valid subdomains in teams pre-known allowlist', async () => {
       const messageOrigin = new URL('https://myteams.microsoft.com');
-      const result = await validateOrigin(messageOrigin);
+      const result = await validateOrigin(messageOrigin, disableCache);
       expect(result).toBe(false);
     });
     it('validateOrigin returns false if first end of origin is not matched valid subdomains in the user supplied list', async () => {
       const messageOrigin = new URL('https://myteams.microsoft.com');
-      const result = await validateOrigin(messageOrigin);
+      const result = await validateOrigin(messageOrigin, disableCache);
       GlobalVars.additionalValidOrigins = ['https://*.teams.microsoft.com'];
       expect(result).toBe(false);
     });
     it('validateOrigin returns false if origin for subdomains does not match in teams pre-known allowlist', async () => {
       const messageOrigin = new URL('https://a.b.sharepoint.com');
-      const result = await validateOrigin(messageOrigin);
+      const result = await validateOrigin(messageOrigin, disableCache);
       expect(result).toBe(false);
     });
     it('validateOrigin returns false if origin for subdomains does not match in the user supplied list', async () => {
       const messageOrigin = new URL('https://a.b.testdomain.com');
-      const result = await validateOrigin(messageOrigin);
+      const result = await validateOrigin(messageOrigin, disableCache);
       GlobalVars.additionalValidOrigins = ['https://*.testdomain.com'];
       expect(result).toBe(false);
     });
   });
+  describe('testing fetch timeout flow', () => {
+    let utils: Utils = new Utils();
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    let timeoutSpy;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    let abortSpy;
+    beforeEach(() => {
+      // Set a mock window for testing
+      utils = new Utils();
+      utils.mockWindow.parent = undefined;
+      app._initialize(utils.mockWindow);
+      GlobalVars.isFramelessWindow = false;
+      jest.useFakeTimers();
+
+      global.AbortController.prototype.abort = jest.fn(() => {
+        throw new Error('AbortError');
+      });
+
+      timeoutSpy = jest.spyOn(global, 'setTimeout');
+      abortSpy = jest.spyOn(AbortController.prototype, 'abort');
+
+      global.fetch = jest.fn(
+        () =>
+          new Promise((resolve) => {
+            jest.advanceTimersByTime(1600);
```

Comment:
>1600 [](http://example.com/codeflow?start=37&length=4)

Can you (export and) reference the `ORIGIN_LIST_FETCH_TIMEOUT_IN_MS` constant here rather than duplicating its value?